src/home/components/HomeCard.js

```diff
@@ -16,20 +16,85 @@
  */
 import React from 'react';
 
-import _ from 'lodash/fp';
+import {
+  Box,
+  Card,
+  CardActions,
+  Divider,
+  Link,
+  Paper,
+  Stack,
+  Typography,
+} from '@mui/material';
 
-import { Card } from '@mui/material';
+import CardActionRouterLink from 'common/components/CardActionRouterLink';
+import RouterButton from 'common/components/RouterButton';
+import RouterLink from 'common/components/RouterLink';
 
-const HomeCard = props => (
+const HomeCard = ({
+  id,
+  title,
+  action,
+  image,
+  children,
+  isDefault,
+  useImg = true,
+  showActionsAsButtons = false,
```

Comment:
question: can there be multiple actions? If not, should this be named `showActionsAsButtons`?